cdc/processor/sourcemanager/engine/pebble/event_sorter.go

```diff
@@ -322,12 +326,16 @@ func (s *EventSorter) Close() error {
 
 	s.mu.RLock()
 	defer s.mu.RUnlock()
-	for tableID, state := range s.tables {
-		if err := s.cleanTable(state, tableID); err != nil {
-			return err
+
+	var err error
+	s.tables.Ascend(func(span tablepb.Span, state *tableState) bool {
```

Comment:
We can left a TODO here, maybe we can use a unified prefix for a given changefeed, so that we can speed up it when closing a changefeed.